image-layout.md

```diff
@@ -2,7 +2,7 @@
 
 The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
 This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
-Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
+Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the [filesystem layers](./layer.md) in the correct order, and then converting the image configuration into an OCI Runtime config.json.
```

Comment:
go build

On Tue, Sep 13, 2016 at 5:23 PM W. Trevor King <notifications@github.com>
wrote:

> In image-layout.md
> <https://github.com/opencontainers/image-spec/pull/305#discussion_r78647065>
> :
>
> > @@ -2,7 +2,7 @@
> >
> >  The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
> >  This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
> > -Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
> > +Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the [filesystem layers](./layer.md) in the correct order, and then converting the image configuration into an OCI Runtime config.json.
>
> On Tue, Sep 13, 2016 at 02:12:47PM -0700, Vincent Batts wrote: > -Given an
> image layout a tool can convert a given ref into a runnable OCI Image
> Format by finding an appropriate manifest from the manifest list, unpacking
> the filesystem serializations in the correct order, and then converting the
> image configuration into an OCI Runtime config.json. > +Given an image
> layout a tool can convert a given ref into a runnable OCI Image Format by
> finding an appropriate manifest from the manifest list, unpacking the
> [filesystem layers](./layer.md) in the correct order, and then converting
> the image configuration into an OCI Runtime config.json. we've been through
> this. This is for explicit relative path. It's not a rule, but many tools
> rely on such to determine that it is a relative path.
> Can you point to such a tool? If we want to require explicit ./, we'll
> want a new PR to update many existing links.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/305/files/7457e6cfb9e8a70e0c0823e7b2ffc24d5b0ca3ab#r78647065>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEF6f8oGVRuL5XudyYAc_obJhwUnV2fks5qpxQ9gaJpZM4J8CpQ>
> .
>